### DISCUSSION TITLE
Description of Image / Default image overlay information

### DISCUSSION THREAD

**Discussion by robs-fotoalbum** [LycheeOrg/Lychee#2134] [Q&A] [answered]
In the Gallery settings the default for image overlays is **desc**

![grafik](https://github.com/LycheeOrg/Lychee/assets/111909427/978f436a-c4a2-4ff4-8f00-a8de1cf981f8)

When you click a photo, then in the detail view the **_title_** is shown together with a part of the EXIF, with the next click on that image a timestamp from the EXIF is displayed, with the next click then you get **_title_** and **_description_** and with the last click only the image is displayed.
Is there any way to control the sequence of what is displayed when?

Additional remarks:
I use IPTC Metadata in Capture One and the field **headline** is set with what will be show in Lychee as **_title_** and **_caption/abstract_** is shown as **_description_** . This seems to be a correct use then, in principle, but the default setting **_desc_** is actually displaying the **_title_**. This is bit confusing, but acceptable.

**Comment by ildyria** [comment, marked as answer]
> Is there any way to control the sequence of what is displayed when?

It is hard-coded:
https://github.com/LycheeOrg/Lychee/blob/master/resources/js/lycheeOrg/flags/photoFlags.ts#L16
https://github.com/LycheeOrg/Lychee/blob/master/resources/views/components/gallery/photo/overlay.blade.php

However there seems to be a bug in the constants (oups)

**Reply by ildyria** [reply]
Fix is on its way: https://github.com/LycheeOrg/Lychee/pull/2135
